From 9dce0ed07b0903c50d818858915266da48829eaa Mon Sep 17 00:00:00 2001
From: Bogdan Purcareata <bogdan.purcareata@nxp.com>
Date: Fri, 9 Feb 2018 15:01:29 +0200
Subject: [PATCH 409/741] staging: fsl-dpaa2/eth: Remove Kconfig dependency on
 QBMAN_DEBUG

commit cd47118713b1ad06cd061a9fd9e6e9e89a94b1f1 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The DPIO implementation no longer uses this Kconfig option. DPAA2 eth
debugfs entries are dependent on this, so remove it.

Signed-off-by: Bogdan Purcareata <bogdan.purcareata@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index dedb0ed0c135..0403f8eacd67 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -31,7 +31,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 
 # QBMAN_DEBUG requires some additional DPIO APIs
 config FSL_DPAA2_ETH_DEBUGFS
-	depends on DEBUG_FS && FSL_QBMAN_DEBUG
+	depends on DEBUG_FS
 	bool "Enable debugfs support"
 	default n
 	---help---
-- 
2.17.1

